### PR TITLE
stylelintrc updates

### PR DESCRIPTION
**Pull request by koskinpark**
This change makes code more readable and logic.

❌Example of incorrect code styling:
```
%a-example {
  margin: 1rem;

  &--with-modifier {
    margin: 5rem;

    @drupal-breakpoint wide_1x {
      margin: 10rem;
    }
  }

  @drupal-breakpoint wide_1x {
    margin: 2rem;
  }
}
```

✔️Now correct code styling 
```
%a-example {
  margin: 1rem;

  @drupal-breakpoint wide_1x {
    margin: 2rem;
  }

  &--with-modifier {
    margin: 5rem;

    @drupal-breakpoint wide_1x {
      margin: 10rem;
    }
  }
}
```

So now `yarn lint-fix` command will re-order your code like that.

### CODE REVIEW

File: starterkits/basic/.storybook/preview-body.html [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<style>
+  body {
+    margin: 20px;
```
Comment (iberdinsky-skilld) [inline starterkits/basic/.storybook/preview-body.html:3]: could it be reason of misunderstanding?
where component starts? and its own padding?

File: starterkits/basic/package.json [hunk 2]
```diff
@@ -45,13 +45,14 @@
...
+    "postcss-discard-empty": "^4.0.1",
     "postcss-extend": "^1.0.5",
     "postcss-import": "^12.0.1",
     "postcss-loader": "^3.0.0",
     "postcss-nested": "^4.1.2",
     "prettier": "^1.16.4",
-    "stylelint": "^13.0.0",
-    "stylelint-config-standard": "^20.0.0",
+    "stylelint": "^13.2.0",
+    "stylelint-config-standard": "^19.0.0",
```
Comment (iberdinsky-skilld) [inline starterkits/basic/package.json:55]: why downgrade?

File: starterkits/basic/.storybook/preview-body.html (hunk 1 above)
Comment (koskinpark) [reply starterkits/basic/.storybook/preview-body.html:3]: well, from my personal experience it is useful thing for storybook and don't break anything even with fixed position (for header) for example. This override created more visual readability of the component

Comment (iberdinsky-skilld) [reply starterkits/basic/.storybook/preview-body.html:3]: but what about pages template. i think it is better to add option to customize it per component?

**Pull request merged**